### PR TITLE
Changed nutrients to return a dict instead of list

### PR DESCRIPTION
**Pull request by bluhmr**
All in all, we have added nutrient support in _schemaorg.py and added several more website scrapers and tests for all.

### CODE REVIEW

File: recipe_scrapers/_schemaorg.py
Comment (bfcarpio) [inline recipe_scrapers/_schemaorg.py:114]: `get()` allows a second parameter of what to return if the key you want doesn't exist. Is this what you want or are you checking if the value of `nutrition` is `Falsey`?

**Review by bfcarpio — CHANGES_REQUESTED** [review]
All good, but just want to clarify some code is why it is.

**Comment by jayaddison** [comment]
Thanks for adding the nutrient retrieval code @bluhmr, that's brilliant :smile: 

Could you add a test to check the dictionary output?  (having test code can be useful for users of the library if they want to figure out what the data should look like, as well as the test helping catch accidental future breakages).

**Comment by bluhmr** [comment]
I'll add a test now, I forgot the test I had made was only local!

**Review by bluhmr — COMMENTED** [review]
In the current version, the false check has been removed

File: recipe_scrapers/_schemaorg.py
Comment (bfcarpio) [inline recipe_scrapers/_schemaorg.py:114]: So if `self.data` doesn't have the key `"nutrition"` then `nutrients` will be `None`. Trying to iterate over `None`, I believe even calling `items()` on it, will cause an error.

If we can be assured that the key will always exist then I don't have a problem with this necessarily. 

I'll approve this and leave it to your judgment.

**Review by bfcarpio — APPROVED** [review]

File: recipe_scrapers/_schemaorg.py
Comment (bluhmr) [reply recipe_scrapers/_schemaorg.py]: I changed the default value to be an empty dict instead in case that does come up.  It is possible that that case would occur.

**Comment by jayaddison** [comment]
> I'll add a test now, I forgot the test I had made was only local!

No worries, thanks for adding that :)

File: tests/test_innit.py
Comment (jayaddison) [inline tests/test_innit.py:61]: I wonder whether the nutrient fieldnames should be 'abstracted' by `recipe-scrapers`, so that `sodiumContent` here (as supplied by the schema.org definition) becomes a field or method named `sodium`, for example?

The reason for that would be that we may later handle additional embedded web data formats, and they may use slightly different key names when describing these nutritional properties.

It'd be nice if users of the library can call `scrape.nutrients().sodium()` and always get the quantity of sodium from the scraped recipe (if it exists), rather than having to check the `nutrients()` dictionary for `sodiumContent`, `sodium`, `sodiumAmount`, etc - depending on what's retrieved today or in future.

That change could be tracked by opening a new issue and doesn't have to be resolved in this pull request, although I think it'd be good to have a plan in mind before merging.

Comment (hhursev) [reply tests/test_innit.py:61]: Definitely an idea for new issue/PRs, yes :) Sounds like a mixin is the way to go for what you propose. A separate mixin-type class that will get updates over time and doesn't impact the core functionality

**Comment by hhursev** [comment]
I have nothing to add except to say thank you 🍻 🤩   Merging and bumping version!

**Pull request merged**
